### PR TITLE
Week 5/project-guess-who

### PR DESCRIPTION
**Pull request by EmmaaBerg**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,64 +1,66 @@
...
+        <option value="brown">brown eyes</option>
+      </optgroup>
+      <optgroup label="accessories">
+        <option value="glasses">glasses</option>
+        <option value="hat">hat</option>
+      </optgroup>
+      <optgroup label="other">
+        <option value="smoker">a smoking habit</option>
+      </optgroup>
+    </select>
```
Comment (Dopest0727) [inline code/index.html:46]: Very organized, easy to read the code. Great job!

File: code/script.js [hunk 2]
```diff
@@ -222,104 +225,128 @@ const generateBoard = () => {
...
-  */
-
-  // Invoke a function to redraw the board with the remaining people.
+    }
+  }
+  // To redraw the board after filtering
+  generateBoard();
+  numberOfGuesses++
+  questionsAsked.innerHTML = `Guesses made: ${numberOfGuesses}`
 }
```
Comment (Dopest0727) [inline code/script.js:321]: Really good overall!

File: code/style.css [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+  background: #f0daf1;
 }
 
 h1 {
   font-size: 42px;
   font-weight: 500;
   line-height: 48px;
   margin: 10px 0;
-  color: white;
+  color: #63458A;
```
Comment (Dopest0727) [inline code/style.css:20]: You can use global variables as in the beginning of the code for a more uniform way of styling.

**Review by Dopest0727 — COMMENTED** [review]
The code is really good Emma! Easy to read and the site works really good! 
Nothing negative to add as i said in the styling, just some preferences of mine and global css variables will change your life! :D

File: code/style.css (hunk 3 above)
Comment (EmmaaBerg) [reply code/style.css:20]: Very good point, I will remember that for the future

**Pull request closed without merging**